### PR TITLE
Link on install

### PR DESCRIPTION
**Pull request by RiccardoRossi**
protype installation with optional linking.

to use it one should do in the configure:

~~~
-DINSTALL_LINK=ON \  ... if set to OFF it copies as usually
-DKRATOS_INSTALL_PREFIX="/path/to/install" \
~~~

this is NOT ready for merge. it is just to discuss the possibility of installing links as outlines in  #5101

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -2,7 +2,6 @@ project (KratosMultiphysics)
 cmake_minimum_required (VERSION 2.8.6)
```
Comment (loumalouomega) [inline CMakeLists.txt:2]: Are we sure?, @philbucher said something about a minimal version

**Review by loumalouomega — COMMENTED** [review]
Just comment

File: CMakeLists.txt (hunk 1 above)
Comment (RiccardoRossi) [reply CMakeLists.txt:2]: i did not change this, but for this to work in windows the minimum is much higher

**Comment by roigcarlo** [comment]
@KratosMultiphysics/technical-committee consider this mechanism implemented in #5170 a very crucial step for the future packaging of Kratos. It would be great to have its problems in windows solved for the next release

**Comment by roigcarlo** [comment]
Closed by #5825

**Pull request closed without merging**